### PR TITLE
nat64-appliance integration in the reproducer

### PR DESCRIPTION
**Pull request by hjensas**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: roles/reproducer/tasks/nat64_appliance.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    path: "{{ cifmw_basedir }}/nat64_appliance"
+    state: directory
+    recurse: true
+    owner: "{{ ansible_user_id }}"
+    group: "{{ ansible_user_gid }}"
+
+- name: Crate SSH keypair for nat64-appliance
+  register: _nat64_key
+  community.crypto.openssh_keypair:
+    comment: "test-key"
```
Comment (cjeanner) [inline roles/reproducer/tasks/nat64_appliance.yml:22]: care to add a better comment? "nat64-access-key" or something?

Comment (hjensas) [reply roles/reproducer/tasks/nat64_appliance.yml]: Done.

**Review by lewisdenny — APPROVED** [review]
/lgtm

File: roles/reproducer/tasks/main.yml [hunk 2]
```diff
@@ -104,6 +104,15 @@
         - bootstrap_env
         - bootstrap_layout
 
+- name: Bootstrap nat64 if needed
+  when:
+    - cifmw_use_libvirt | default(false) | bool
+    - cifmw_use_nat64 | default(false) | bool
+  tags:
+    - bootstrap_nat64
```
Comment (cjeanner) [inline roles/reproducer/tasks/main.yml:112]: that should probably be part of `bootstrap_layout` and `bootstrap_env`

File: roles/reproducer/tasks/main.yml [hunk 3]
```diff
@@ -104,6 +104,15 @@
...
+  ansible.builtin.include_tasks:
+    file: nat64_appliance.yml
```
Comment (cjeanner) [inline roles/reproducer/tasks/main.yml:114]: When using `include_tasks` you want to also `apply` tags:

```
ansible.builtin.include_tasks:
  file: nat64_appliance.yml
  apply:
    tags:
      - bootstrap_nat64
      - bootstrap_layout
      - bootstrap_env
```
Else it won't do any task if you use `--tags bootstrap_nat64`

**Review by cjeanner — DISMISSED** [review]

File: roles/reproducer/tasks/main.yml (hunk 2 above)
Comment (hjensas) [reply roles/reproducer/tasks/main.yml:112]: Done

File: roles/reproducer/tasks/main.yml (hunk 3 above)
Comment (hjensas) [reply roles/reproducer/tasks/main.yml:116]: Done

**Review by raukadah — APPROVED** [review]

**Pull request merged**